dataprofiler/labelers/column_name_model.py

```diff
@@ -39,7 +39,10 @@ def __init__(self, parameters=None):
         parameters.setdefault("include_label", True)
         parameters.setdefault("negative_threshold_config", None)
 
-        # initialize class
+        label_mapping = {"not": "implemented"}
+        self.set_label_mapping(label_mapping)
```

Comment:
FEEDBACK: I don't like and it is very spaghetti code to force the labeler `load_with_components` to run. The API on the labeler requires a `label_mapping`; however, for this model, we don't use a label mapping **currently**. We could refactor the code to ultimately use that but POC / MVP stage the data or the model aren't really setup to use this attribute of the API.